buildpack/start.py

```diff
@@ -27,7 +27,7 @@
 from buildpack.runtime_components import security
 from lib.m2ee import M2EE as m2ee_class
 
-BUILDPACK_VERSION = "4.5.7"
+BUILDPACK_VERSION = "4.6.0"
```

Comment:
I think that because the 4.6.0 tag was already created in the previous release, we need to go to 4.6.1 here.

Reply:
Indeed we need to go to v4.6.1, re-using tags in git is messy.
Also, we should re-consider adding the version number in code, maybe generating from a git ref / tag during the build process is the way to go here.

Reply:
I've bumped the version to 4.6.1